metadata.json

```diff
@@ -1,4 +1,22 @@
 {
+  "name": "puppet-caddy",
+  "version": "1.0.1-rc0",
+  "author": "Vox Pupuli",
+  "summary": "Puppet Caddy Module",
+  "license": "MIT",
+  "source": "https://github.com/voxpupuli/puppet-caddy",
+  "issues_url": "https://github.com/voxpupuli/puppet-caddy/issues",
+  "dependencies": [
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= 4.0.0 < 5.0.0"
```

Comment:
If I introduce a new dependency, I always add the last release as lower bound because thats the version where we know it works. That would be 4.4.0 for archive. But I don't think that there is an official recommendation in the style guide.

Reply:
I was just looking for the first Puppet 5+ only release. But 4.3.0 should be ok, because it added support for CentOS 8.

Reply:
I just saw that 4.4.0 fixed an issue related to cookies / proxies and fixed the lower boundary.